changelogs/unreleased/gh-7972-func-def-wrong-options-err-msg.md

```diff
@@ -1,4 +1,4 @@
 ## bugfix/box
 
-* Fixed error message for wrong options provided to function definition
+* Fixed the error message for wrong options provided to a function definition
   (gh-7972).
```

Comment:
It is hard to deduced what is a function of the question. However, it is obvious from #7972. How about writing something like `box.func function definition`?